src/components/homepage/HeroSection.jsx

```diff
@@ -33,197 +31,169 @@ const EditorHeroSection = ({
+      <FormTitle>Hero title</FormTitle>
+      <FormField
+        placeholder="Hero title"
+        id={`section-${sectionIndex}-hero-title`}
+        value={title}
+        onChange={onFieldChange}
+      />
+      <FormError>{errors.sections[0].hero.title}</FormError>
+    </FormContext>
+    <FormContext isRequired hasError={!!errors.sections[0].hero.subtitle}>
+      <FormTitle>Hero subtitle</FormTitle>
+      <FormField
+        placeholder="Hero subtitle"
+        id={`section-${sectionIndex}-hero-subtitle`}
+        value={subtitle}
+        onChange={onFieldChange}
+      />
+      <FormError>{errors.sections[0].hero.subtitle}</FormError>
+    </FormContext>
+    <FormContext
+      hasError={!!errors.sections[0].hero.background}
+      onFieldChange={onFieldChange}
+      isRequired
+    >
+      <FormTitle>Hero background image</FormTitle>
+      <FormFieldMedia
+        value={background}
+        id={`section-${sectionIndex}-hero-background`}
+        inlineButtonText="Select"
+      />
+      <FormError>{errors.sections[0].hero.background}</FormError>
+    </FormContext>
+    <div className={styles.card}>
+      <p className={elementStyles.formLabel}>Hero Section Type</p>
+      {/* Permalink or File URL */}
+      <div className="d-flex">
+        <label htmlFor="radio-highlights" className="flex-fill">
+          <input
+            type="radio"
+            id="radio-highlights"
+            name="hero-type"
+            defaultValue="highlights"
+            onChange={handleHighlightDropdownToggle}
+            checked={!dropdown}
           />
-          <FormError>{errors.sections[0].hero.title}</FormError>
-        </FormContext>
-        <FormContext isRequired hasError={!!errors.sections[0].hero.subtitle}>
-          <FormTitle>Hero subtitle</FormTitle>
-          <FormField
-            placeholder="Hero subtitle"
-            id={`section-${sectionIndex}-hero-subtitle`}
-            value={subtitle}
-            onChange={onFieldChange}
+          Highlights + Button
+        </label>
+        <label htmlFor="radio-dropdown" className="flex-fill">
+          <input
+            type="radio"
+            id="radio-dropdown"
+            name="hero-type"
+            defaultValue="dropdown"
+            onChange={handleHighlightDropdownToggle}
+            checked={!!dropdown}
           />
-          <FormError>{errors.sections[0].hero.subtitle}</FormError>
-        </FormContext>
-        <FormContext
-          hasError={!!errors.sections[0].hero.background}
-          onFieldChange={onFieldChange}
-          isRequired
-        >
-          <FormTitle>Hero background image</FormTitle>
-          <FormFieldMedia
-            value={background}
-            id={`section-${sectionIndex}-hero-background`}
-            inlineButtonText="Select"
+          Dropdown
+        </label>
+      </div>
+      <span className={elementStyles.info}>
+        Note: you can only have either Key Highlights+Hero button or a Hero
```

Comment:
are we missing a space here for `Highlights+Hero`?

Reply:
not doing this - this whole component gets removed later on and this text doesn't exist anymore in figma (see [here](https://www.figma.com/file/7OtNYkc4s9BkFk05ConEbd/%E2%AD%90-V1.5?type=design&node-id=12732-119415&mode=dev)). 

also note that this missing space is present on prod too, see ss below
![Uploading Screenshot 2023-08-17 at 5.31.11 PM.png…]()